src/wrapHandlers.mjs

```diff
@@ -28,7 +28,7 @@ export function wrapNonWebsocket(fn) {
     if (WebSocketWrapper.isInstance(res)) {
       next('route');
     } else {
-      fn(req, res, next);
+      (async () => fn(req, res, next))().catch(next);
```

Comment:
I wonder if this simpler option would also solve your issue?

```js
return fn(req, res, next);
```

(I assume the `catch -> next` is already being handled by express at a higher level, so this simplification should avoid 2 microtasks)

Reply:
yeah thatll work two!
commited 👍